src/shared/components/case-list-filters/case-list-filters.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { READ_ACCESS } from '../../domain/case-view/access-types.model';
+import { DefinitionsService } from '../../services';
+import { Jurisdiction } from '../../domain';
+
+@Component({
+  selector: 'ccd-case-list-filters',
+  templateUrl: './case-list-filters.component.html'
+})
+export class CaseListFiltersComponent implements OnInit {
```

Comment:
Unit tests checking emitting apply and reset notifications?

Reply:
added tests